### PR TITLE
TreeView, List - If the JAWS screen reader is used, keyboard navigation doesn't work (T747286)

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**
The reader doesn't pronounce the item text when a search bar is enabled

### CODE REVIEW

**Review by jsdmitry — DISMISSED** [review]

File: js/ui/widget/ui.search_box_mixin.js [hunk 1]
```diff
@@ -146,10 +150,19 @@ module.exports = {
         return searchMode === "equals" ? "=" : searchMode;
     },
 
+    _cleanAria: function($target) {
+        this.setAria({
+            "role": undefined,
```
Comment (DokaRus) [inline js/ui/widget/ui.search_box_mixin.js]:
```suggestion
            "role": null,
```

File: js/ui/widget/ui.search_box_mixin.js [hunk 2]
```diff
@@ -146,10 +150,19 @@ module.exports = {
...
+            "activedescendant": undefined
```
Comment (DokaRus) [inline js/ui/widget/ui.search_box_mixin.js]:
```suggestion
            "activedescendant": null
```

File: js/ui/widget/ui.search_box_mixin.js [hunk 3]
```diff
@@ -146,10 +150,19 @@ module.exports = {
...
+        }, $target);
+        domUtils.toggleAttr($target, "tabIndex");
```
Comment (DokaRus) [inline js/ui/widget/ui.search_box_mixin.js]: Pull 19.1 branch. I removed this method some time ago.
Use `$.attr()` instead of it

**Review by DokaRus — CHANGES_REQUESTED** [review]

**Review by DokaRus — APPROVED** [review]
LGTM

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**